plugin-server/src/cdp/hog-masker.ts

```diff
@@ -1,6 +1,6 @@
 import { createHash } from 'crypto'
 
-import { execHog } from './hog-executor'
```

Comment:
This is also a service so if we just called them `services` we could encapsulate them all together